Data store (#117)
HumanCompatibleAI/overcooked_ai: src/overcooked_demo/README.md

### 1): Using Docker Compose CLI to deploy on EC2
With the Overcooked demo, you can test the interaction between two human players. To do this, you need to deploy this code on a server (https://docs.docker.com/language/python/deploy/). 

> After successful deployment, the first user (User 1) should go to http://[server_ip_address]/, select the human keyboard input for both players, and select the setting for this game (e.g. whether to collect data or to run the game with old dynamics, which means the soup starts cooking automatically when all ingredients are received), and click "Create game". If everything has been successful, he will receive a message: "Waiting for game to start". 

Reviewer: ```suggestion
After successful deployment, the first user (User 1) should go to http://[server_ip_address]/, select the human keyboard input for both players, and select the setting for this game (e.g. whether to collect data and/or to run the game with new dynamics, which means the soup requires an additional "interact" action to start cooking), and click "Create game". If everything has been successful, he will receive a message: "Waiting for game to start". 
```

```

7: Once the server is up, you can access the game through your browser by passing in the public ip of your server and port 80.

### 3): Running predefined experiments

One can also define a series of experiments to be run when players join the game. To do that, go to [`config.json`](./server/config.json) and modify the "predefined" parameters by changing what the layouts you want to include and the settings for them. 

> Save the changes and spin up a server by ways described in the earlier sections. User 1 should go to http://[server_ip_address]/predefined, and he/she should be placed in a waiting room, and he will see how many layouts are there in the current series of experiments. User 2 can now go to http://[server_ip_address]/predefined, and the game should automatically starts. The games will be run back to back until it goes through all the predefined layouts in the config file.

Reviewer: ```suggestion
Save the changes and spin up a server by ways described in the earlier sections. User 1 should go to http://[server_ip_address]/predefined, where they should be placed in a waiting room, and they will see how many layouts are there in the current series of experiments. User 2 can now go to http://[server_ip_address]/predefined, and the game should automatically start. The games will be run back to back until it goes through all the predefined layouts in the config file.
```
